docs/curl-recipes/01-health-system-status.md

```
@@ -0,0 +1,98 @@
+# Health & System Status
+
+Basic health checks and system status monitoring.
+
+## Simple Health Check
+
+```bash
+curl -s "$API_BASE/health"
+```
+
+Expected response:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)